lib/src/Uncertainty/Distribution/MeixnerDistribution.cxx

```diff
@@ -402,6 +386,75 @@ class MeixnerBoundCD: public FunctionImplementation
   const DistributionImplementation::Implementation p_distribution_;
 }; // class MeixnerBoundCD
 
+// Here we inherit from EvaluationImplementation in order to benefit from the inherited evaluation over a sample
+class MeixnerPDFGradientWrapper: public EvaluationImplementation
+{
+public:
+  MeixnerPDFGradientWrapper(const DistributionImplementation::Implementation & p_distribution)
+    : EvaluationImplementation()
+    , p_distribution_(p_distribution)
+  {
+    // Nothing to do
+  }
+
+  MeixnerPDFGradientWrapper * clone() const
+  {
+    return new MeixnerPDFGradientWrapper(*this);
+  }
+
+  Point operator() (const Point & point) const
+  {
+    return p_distribution_->computePDFGradient(point);
+  }
+
+  UnsignedInteger getInputDimension() const
+  {
+    return 1;
+  }
+
+  UnsignedInteger getOutputDimension() const
+  {
+    return 4;
+  }
+
```

Comment:
no need to overload description methods and str/repr (since the function is not exposed to user)
only clone/getIn/Outputdimension are strictly needed

Reply:
also they should be marked override

Reply:
These methods were here for debug purpose. I will remove them.